go/runtime/bundle/manifest.go

```diff
@@ -148,6 +151,38 @@ func (c ComponentID) String() string {
 	return string(c.Kind) + " (" + c.Name + ")"
 }
 
+// MarshalText serializes the component identifier into text form.
+func (c ComponentID) MarshalText() ([]byte, error) {
+	return []byte(string(c.Kind) + kindNameSeparator + c.Name), nil
```

Comment:
I would expect that the component identifier is unique, but is not as `"ronl."` and `"ronl"` unmarshal the same way.

Reply:
Fixed.